internal/pkg/api/server/common/http/decode.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 // decider decodes request.
-func newDecoder() func(r *http.Request) goaHTTP.Decoder {
```

Comment:
Některý věci nepotřebovaly velké změny na to, aby se daly použít pro obě API, tak jsem je přesunul do `common`.

Reply:
:+1: